SSH key import fails with a provider panic

When attempting to import an SSH key per the docs:

`terraform import juju_ssh_key.stg-jsimpso-test stg-jsimpso-test` 

The provider quits in a panic:
```
╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵


Stack trace from the terraform-provider-juju_v0.5.0 plugin:

panic: runtime error: index out of range [1] with length 1

```
Looking at the provider code vs the documentation, it seems this is a documentation issue:

https://github.com/juju/terraform-provider-juju/blob/3576d763e6ee6f7cd00ee27a37e381b112d722ce/internal/provider/resource_ssh_key.go#L80-L83

https://github.com/juju/terraform-provider-juju/blob/3576d763e6ee6f7cd00ee27a37e381b112d722ce/examples/resources/juju_ssh_key/import.sh#L1-L2

Confirmed this by successfully importing an SSH key here:
```
terraform import juju_ssh_key.stg-jsimpso-test blank:stg-jsimpso-test:stg-jsimpso-test
...
Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform import juju_ssh_key.username <username>`
2. `terraform import juju_ssh_key.username <anything>:<model_name>:<username>`

--- a/docs/resources/ssh_key.md
+++ b/docs/resources/ssh_key.md
@@ -36,6 +36,6 @@ resource "juju_ssh_key" "mykey" {
 Import is supported using the following syntax:
 
 ```shell
-# Keys can be imported with the username of the key
-$ terraform import juju_ssh_key.dev-user dev-user
+# Keys can be imported with the name of the model and the username of the key
+$ terraform import juju_ssh_key.dev-user ssh_key:development:dev-user
 ```

--- a/examples/resources/juju_ssh_key/import.sh
+++ b/examples/resources/juju_ssh_key/import.sh
@@ -1,2 +1,2 @@
-# Keys can be imported with the username of the key
-$ terraform import juju_ssh_key.dev-user dev-user
+# Keys can be imported with the name of the model and the username of the key
+$ terraform import juju_ssh_key.dev-user ssh_key:development:dev-user

